Tests/FluentAssertions.Specs/Specialized/TaskOfTAssertionSpecs.cs

```diff
@@ -143,6 +143,28 @@ public async Task When_task_completes_late_it_should_fail()
             await action.Should().ThrowAsync<XunitException>();
         }
 
+        [Fact]
+        public async Task When_task_completes_late_it_in_AssertionScope_should_fail()
```

Comment:
⛏️ Please don't use the literal names of types in test names. Keep them functional.

Reply:
done